frontend/src/assets/css/variables.scss

```diff
@@ -1,49 +1,97 @@
 :root {
-    // Error colors
-    --error-main: #ff2121;
+    // *Basic Colors*
 
-    // Action colors
-    --action-hover: #2e61ae;
+    // Red
+    --torch-red: #ff2121;
+
+    // Azure
+    --azure: #2e61ae;
+
+    // White
+    --white: #ffffff;
+    --white-lilac: #f7f8fc;
+
+    // St Tropaz
+    --st-tropaz: #274f8d;
+
+    // Gray
+    --steel-gray: #1f1e29;
+    --slate-gray: #7c8798;
+
+    // Nepal
+    --nepal: #90a3bf;
+
+    // Botticelli
+    --botticelli: #d5dce8;
+
+    // Solitude
+    --solitude: #dce5ff;
+
+    // Mountain Meadow
+    --mountain-meadow: #12b76a;
+
+    // Porcelain
+    --porcelain: #f5f6f7;
+
+    // *Reusable Elements*
 
     // Background colors
-    --background-primary: #f7f8fc;
-    --background-secondary: #274f8d;
+    --background-primary: var(--white-lilac);
+    --background-secondary: var(--st-tropaz);
 
     // Text colors
-    --text-primary-dark: #1f1e29;
-    --text-primary: #90a3bf;
-    --text-secondary: #7c8798;
+    --text-primary: var(--nepal);
+    --text-primary-dark: var(--steel-gray);
+    --text-secondary: var(--slate-gray);
+
+    // Action colors
+    --hover-primary: var(--azure);
+
+    // Shadow colors
+    --shadow-primary: #2e61ae65;
+    --shadow-secondary: #10161a33;
+
+    // Error colors
+    --error-primary: var(--torch-red);
+
+    // *Components*
 
     // Input colors
-    --input-main: #d5dce8;
+    --input-border: var(--botticelli);
 
     // Tag colors
-    --tag-main: #dce5ff;
-
-    // Colors
-    --bt-blue-gray: #90a3bf;
-    --bt-input: #d5dce8;
-    --hover: #274f8d;
-    --light-background-color: #f7f8fc;
-    --hover-background-color: #f7f8fc;
-    --disabled-background-color: #90a3bf;
-    --hover-ripple-color: #2e61ae65;
-    --shadow-color: rgb(16 22 26 / 20%);
-    --checked-background-color: #274f8d;
-    --checked-inner-background-color: #f7f8fc;
-    --checkbox-border: #d5dce8;
-    --checkbox-disabled: #f7f8fc;
-    --checkbox-checked: #274f8d;
-    --checkbox-disabled-checked: #d5dce8;
-    --header-border: #f5f6f7;
-    --status-online: #12b76a;
-    --status-offline: #90a3bf;
-    --white-color: #ffffff;
-
-    /* Font family. */
+    --tag-background: var(--solitude);
+
+    // Checkbox colors
+    --checked-background: var(--background-secondary);
+    --checked-inner-background: var(--background-primary);
```

Comment:
we can remove it

Reply:
the specific variable can be on the component level. here we need universal variables

Reply:
I removed all component variables